### ISSUE TITLE
Hope to see a release on 1.12.2 :D

### ISSUE THREAD

**Issue by yudonglin** [PinkaLulan/ShinColle#57]
meet some problems with API...
Does this mod require baubles and metamorph on 1.12.2 version?

**Comment by Simba98** [comment]
Hoping too.
But now is 2019, well, maybe we can't see it forever.

**Comment by yudonglin** [comment]
> Hoping too.
> But now is 2019, well, maybe we can't see it forever.

Actually... Someone did make a 1.12.2 release...

**Comment by Simba98** [comment]
> Hoping too.
> But now is 2019, well, maybe we can't see it forever.
> 
> Actually... Someone did make a 1.12.2 release...

Yes I found it, but it still have some bugs, maybe due to the changes of forge APIs in 1.12.x.

**Comment by yudonglin** [comment]
> > Hoping too.
> > But now is 2019, well, maybe we can't see it forever.
> > Actually... Someone did make a 1.12.2 release...
> 
> Yes I found it, but it still have some bugs, maybe due to the changes of forge APIs in 1.12.x.

I will try to do a mcreator version if I have time

**Comment by 71USERX** [comment]
Was this mod forgotten by author? (Minecraft 1.10.2 was released in 2016. :D)
